doc/api/deprecations.md

```diff
@@ -3047,6 +3047,20 @@ const w = new Writable({
 });
 ```
 
+### DEP0XXX: Thenable support in streams
+
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/40773
+    description: Runtime deprecation.
+-->
+
+Type: Runtime
```

Comment:
This was discussed in TSC meeting today, no objections was raised to skip the runtime deprecation.
```suggestion
    description: End-of-life.
-->

Type: End-of-Life
```